source/install/self-managed-changelog.md

```diff
@@ -6,11 +6,171 @@ See the [changelog in progress](https://bit.ly/2nK3cVf) for the upcoming release
 
 Latest Mattermost Releases:
 
+- [Release v7.8 - Extended Support Release](#release-v7-8-extended-support-release)
 - [Release v7.7 - Feature Release](#release-v7-7-feature-release)
 - [Release v7.5 - Feature Release](#release-v7-5-feature-release)
 - [Release v7.4 - Feature Release](#release-v7-4-feature-release)
 - [Release v7.1 - Extended Support Release](#release-v7-1-extended-support-release)
 
+## Release v7.8 - [Extended Support Release](https://docs.mattermost.com/upgrade/release-definitions.html#extended-support-release-esr)
+
+**Release Day: February 16, 2023**
+
+### Important Upgrade Notes
+
+Before upgrading, we recommend checking for duplicate data in the ``focalboard_category_boards`` table, and deleting all but one copy of duplicate data. This is to ensure that the new plugin version startup doesn't lock the table, and prevent users from using Boards. We recommend de-duplicating the data at a time of low user activity.
```

Comment:
cc @harshilsharma63 @sbishel A question from the GitLab team:

"is there a documented fix for if this occurs? While we can add this to our release notes, it seems likely that we will have users run into this."

https://gitlab.com/gitlab-org/omnibus-gitlab/-/issues/7411#note_1271174901

Reply:
So by design the table shouldn't have any duplicate data. Though on my local system and about 2-4 times while testing, I've seen the table having just a few records of duplicate rows. So this is a precautionary check rather than a data-cleanup. v7.8 is on community as well and we never ran into the issue.

I'm not sure what we mean by a "fix" here. @sbishel any recommendation?